catalogs/sources/gtfs/schedule/br-sao-paulo-sao-paulo-transporte-sptrans-gtfs-8.json

```diff
@@ -0,0 +1,21 @@
+{
+    "mdb_source_id": 8,
+    "data_type": "gtfs",
+    "provider": "S\u00e3o Paulo Transporte (SPTrans)",
+    "location": {
+        "country_code": "BR",
+        "subdivision_name": "S\u00e3o Paulo ",
+        "municipality": "S\u00e3o Paulo ",
```

Comment:
@maximearmstrong 

- It looks like accents are not supported right now: S\u00e3o Paulo
- There are some blank spaces that are being included in strings - is this something we can clean for in the code or should I do this in the import CSV?

Reply:
Right, I will open an issue for the accents. For the blank spaces, my initial thoughts were that the operations would add the input given by the user as is. I will do the modification in the script so you don't have to modify the CSV.